src/components/Superfest/NFTClaimingBox/NFTClaimingBox.style.ts

```diff
@@ -88,7 +88,7 @@ export const NFTDisplayBox = styled(Box)(({ theme }) => ({
   flexWrap: 'wrap',
   justifyContent: 'center',
   alignContent: 'center',
-  gap: '48px',
+  gap: '16px',
```

Comment:
```suggestion
  theme.spacing(2)
```

Reply:
I have a task to clean all the theme.spacing.